src/worker/plugins/run.ts

```diff
@@ -58,7 +58,7 @@ export async function runProcessEvent(server: PluginsServer, event: PluginEvent)
 
             try {
                 returnedEvent = (await processEvent(returnedEvent)) || null
-                if (returnedEvent.team_id != teamId) {
+                if (returnedEvent?.team_id != teamId) {
```

Comment:
First, in JS we should always use the `!==` and `===` operators instead of `!=` and `==`. The shorter ones equate all falsy fields. Basically, `"" == false == 0`, but `"" !== false !== 0`. 

Then, if `returnedEvent` is null (plugin skipped it) and hence its `team_id` is undefined, this will throw the error.